### PR TITLE
Updated dockerfile to respect changes in regular dockerfile

### PR DESCRIPTION
**Pull request by yuraku**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -46,34 +80,32 @@ RUN dnf config-manager --disable && \
...
-# Prepare Management Center
-RUN wget -O ${MC_HOME}/${MC_INSTALL_ZIP} \
-          http://download.hazelcast.com/management-center/${MC_INSTALL_ZIP} \
- && unzip ${MC_INSTALL_ZIP} \
-      -x ${MC_INSTALL_NAME}/docs/* \
- && rm -rf ${MC_INSTALL_ZIP} \
- && mv ${MC_INSTALL_NAME}/* . \
- && rm -rf ${MC_INSTALL_NAME}
+RUN echo "Installing new packages" \
+    && dnf -y --setopt=tsflags=nodocs install java-11-openjdk &> /dev/null \
```
Comment (leszko) [inline Dockerfile:84]: Why is this `dnf` repeated here again. I guess you already executed that in the line 80.

File: Dockerfile [hunk 2]
```diff
@@ -1,20 +1,59 @@
-FROM registry.access.redhat.com/ubi8/ubi
-MAINTAINER Hazelcast, Inc. Management Center Team <info@hazelcast.com>
+ARG MC_VERSION=4.2020.12
+ARG MC_INSTALL_NAME="hazelcast-management-center-${MC_VERSION}"
+ARG MC_INSTALL_JAR="hazelcast-management-center-${MC_VERSION}.jar"
 
-ENV MC_VERSION 4.2020.10
-ENV MC_HOME /opt/hazelcast/management-center
-ENV MC_DATA /data
+FROM alpine:3.12.1 AS builder
```
Comment (hasancelik) [inline Dockerfile:5]: I could not find any info about init containers at [RedHat Certification documentation](https://redhat-connect.gitbook.io/partner-guide-for-red-hat-openshift-and-container/program-on-boarding/certification-workflow) but I have found ubi8 based init container [registry.access.redhat.com/ubi8/ubi-init](https://catalog.redhat.com/software/containers/ubi8/ubi-init/5c359b97d70cc534b3a378c8?container-tabs=overview&gti-tabs=unauthenticated)

I think we should ask to RedHat team, can we use non-RHEL init container or should we use above one?

**Review by leszko — APPROVED** [review]
Added one comment. Other than that, LGTM 👍 

Remember to check if it passes the Red Hat Connect scanner.

**Review by hasancelik — APPROVED** [review]
LGTM, just one minor comment needs to be clarified before the merge 👍

**Review by emre-aydin — APPROVED** [review]

**Comment by yuraku** [comment]
> Added one comment. Other than that, LGTM 👍
> 
> Remember to check if it passes the Red Hat Connect scanner.

Could you please clarify on how I can check that? Have never tried this before

File: Dockerfile (hunk 1 above)
Comment (yuraku) [reply Dockerfile:84]: removed, thanks. leftover from c/p

**Comment by hasancelik** [comment]
@yuraku you can create an image at your local and push it to Redhat registry, scan job will start automatically. After build finished, you can remove it.

File: Dockerfile (hunk 2 above)
Comment (leszko) [reply Dockerfile:5]: I think that it's just a build image, so we can use whatever we want. Also if it passes the Red Hat scan, then I think it's fine.

**Review by leszko — APPROVED** [review]

**Pull request merged**